# the arguments that will be received by the runned script
script_arguments: ['argument1', 'argument2']
